app/components/organization/integrations/slack/index.ts

```
@@ -0,0 +1,255 @@
+
+      this.notify.success(this.tSlackWillBeRevoked);
+    } catch (error) {
+      this.notify.error(this.tPleaseTryAgain);
+    }
+  });
+
+  integrateSlack = task(async (changeset) => {
+    await changeset.validate();
+
+    if (!changeset.isValid) {
+      if (changeset.errors && changeset.errors[0].validation) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)